buildkit/packaging/opensuse.py

```diff
@@ -38,25 +70,41 @@ def generate_packaging(config_bundle, output_dir, build_output=DEFAULT_BUILD_OUT
     Raises FileExistsError if output_dir already exists and is not empty.
     Raises FileNotFoundError if the parent directories for output_dir do not exist.
     """
+
+    ensure_empty_dir(output_dir) # Raises FileNotFoundError, FileExistsError
+    (output_dir / 'scripts').mkdir()
+    (output_dir / 'archive_include').mkdir()
+    
+    # Patches
+    config_bundle.patches.export_patches(output_dir / PATCHES_DIR)
+    
+    patchInfo = _get_spec_format_patch_series(output_dir / PATCHES_DIR / 'series')
+
     build_file_subs = dict(
         build_output=build_output,
+        unpacked_src_dir=_get_absolute_tree_path(),
+        gn_flags=_get_parsed_gn_flags(config_bundle.gn_flags),
         gn_args_string=' '.join(
             '{}={}'.format(flag, value) for flag, value in config_bundle.gn_flags.items()),
+        numbered_patch_list=patchInfo['patchString'],
+        apply_patches_cmd=_get_patch_apply_spec_cmd(patchInfo['numPatches']),
         version_string=config_bundle.version.version_string
     )
 
-    ensure_empty_dir(output_dir) # Raises FileNotFoundError, FileExistsError
-    (output_dir / 'scripts').mkdir()
-    (output_dir / 'archive_include').mkdir()
-
     # Build and packaging scripts
     _copy_from_resources('build.sh.in', output_dir)
-    _copy_from_resources('package.sh.in', output_dir)
+    _copy_from_resources('setup.sh.in', output_dir)
+    _copy_from_resources('ungoogled-chromium.spec.in', output_dir)
     _copy_from_resources(PROCESS_BUILD_OUTPUTS, output_dir / 'scripts', shared=True)
     process_templates(output_dir, build_file_subs)
 
     # Other resources to package
     _copy_from_resources('README', output_dir / 'archive_include')
+    _copy_from_resources('master_preferences', output_dir)
+    _copy_from_resources('chromium-browser.sh', output_dir)
+    _copy_from_resources('chromium-browser.desktop', output_dir)
+    _copy_from_resources('chromium-browser.xml', output_dir)
+    _copy_from_resources('chromium.default', output_dir)
+    _copy_from_resources('chromium-browser.appdata.xml', output_dir)
+    _copy_from_resources('chromium-icons.tar.bz2', output_dir)
```

Comment:
Please replace the `.tar.bz2` with its contents (code to generate the archive is proposed in other comment for `setup.sh`). I like to keep files reasonably transparent in the repo.